engine/battle/ai/scoring.asm

```diff
@@ -1230,8 +1264,10 @@ AI_Smart_HyperBeam:
 	dec [hl]
 	ret
 
-.discourage
-; If enemy's HP is above 50%, discourage this move at random
+.randomly_discourage
+; If enemy's HP is above 50%, discourage this move as follows :
+; - 42% discourage 
+; - 42% greatly discourage
```

Comment:
The other 14%?

Reply:
When i don't include remaining percentages, that means nothing happens. For the sake of completeness, i could write it out clearly.